packages/matchbox/src/components/TextField/TextField.js

```diff
@@ -139,16 +139,18 @@ function TextField(props) {
 
   return (
     <StyledWrapper {...systemProps}>
-      <Label id={id} label={label} labelHidden={labelHidden}>
-        {required && (
-          <Box as="span" pr="200" aria-hidden="true">
-            *
-          </Box>
-        )}
-        {error && errorInLabel && (
-          <Box as={Error} id={errorId} wrapper="span" error={error} fontWeight="400" />
-        )}
-      </Label>
+      {label && (
```

Comment:
Looks like Selects could use this too. Would it make more sends to add `if (!label) return null` in the `Label` component? Up to you here

Reply:
that's a good idea!

Reply:
for the record, it looks like Select was already handling this.